### PR TITLE
Feature/view docs

### PR DESCRIPTION
**Pull request by jurgenwerk**

### CODE REVIEW

File: source/endpoints/views.md.erb [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+filter_groups  | Filter configuration
+direction | The direction to sort by. <br><br>Possible values: asc, desc
+search    | A general search string to filter by keywords' term, tags, result URL and location
+page      | Page number (pagination)
+group_id  | Group ID
+scope     | when group ID is not provided, this value has to be "account"
```
Comment (emnlvrz) [inline source/endpoints/views.md.erb]: Description should start in capitals :smile:

File: source/endpoints/views.md.erb [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+
+             Condition            |
+----------------------------- |
+less |
+more |
+less_or_equal |
+more_or_equal |
+equals |
+
+### Presence and absence conditions
```
Comment (emnlvrz) [inline source/endpoints/views.md.erb:60]: Should we add some words about to which fields this applies? Even if they apply to every  field, maybe? :thinking:

File: source/endpoints/views.md.erb [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+
+### HTTP Request
+
+`GET https://api.nightwatch.io/api/v1/dynamic_views?access_token=ACCESS_TOKEN`
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+group_id  | views belonging to a specific group
```
Comment (emnlvrz) [inline source/endpoints/views.md.erb]: Capitals :point_up:

File: source/endpoints/views.md.erb [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+
+### URL Parameters
+
+Parameter | Description
+--------- | -----------
+dynamic_view_id* | View's ID
+sort      | The name of the field to sort by. <br><br>Possible values: id, query, google_gl, google_hl, last_processed_at, created_at, updated_at, position, position_type, position_organic, position_places, position_places_image, position_changed_by, position_changed_status, position_changed_at, position_local_pack, position_featured_snippet, position_knowledge_panel, local_search, position_changed_during_last_day, position_changed_during_last_week, position_changed_during_last_month, mobile, engine, previous_position, best_position, available_serp_features, adwords_global_search_volume, adwords_global_average_cpc, adwords_local_search_volume, adwords_local_average_cpc
+direction | The direction to sort by. <br><br>Possible values: asc, desc
+page      | Page number (pagination)
+scope     | when group ID is not provided, this value has to be "account"
```
Comment (emnlvrz) [inline source/endpoints/views.md.erb]: Capitals

**Review by emnlvrz — CHANGES_REQUESTED** [review]
This API kinda sucks, specially the filter groups part. I would suggest adding *Get Filter Groups* and *Update Filter Groups* sections (like we do in *Create a view* section). Unless we plan to do the filter groups endpoint separately.

File: source/endpoints/views.md.erb (hunk 2 above)
Comment (jurgenwerk) [reply source/endpoints/views.md.erb:60]: Yes. Good idea. Will add it.

File: source/endpoints/views.md.erb (hunk 1 above)
Comment (jurgenwerk) [reply source/endpoints/views.md.erb]: You're so meticulous 😍

File: source/endpoints/views.md.erb (hunk 3 above)
Comment (jurgenwerk) [reply source/endpoints/views.md.erb]: Yesh

**Pull request merged**